Bottom: dca360a9c6efc262e74b1882cfb6fd56c6a1ce7f
Top:    e05055862da9c7b7ccb85cefb2f1221fab98204f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 02:19:47 +0000

Refresh of add-sl_rats-layer-id-to-tell-t

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 93bb2b8..6ff7e74 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -2890,6 +2890,8 @@ lesstif_set_layer (const char *name, int group, int empty)
 	case SL_UDRILL:
 	case SL_PDRILL:
 	  return 1;
+	case SL_RATS:
+	  return PCB->RatOn;
 	}
     }
   return 0;
